src/ncp-spinel/SpinelNCPInstance.cpp

```diff
@@ -1964,6 +1964,9 @@ SpinelNCPInstance::regsiter_all_get_handlers(void)
 	register_get_handler_spinel_simple(
 		kWPANTUNDProperty_OpenThreadLogLevel,
 		SPINEL_PROP_DEBUG_NCP_LOG_LEVEL, SPINEL_DATATYPE_UINT8_S);
+	register_get_handler_spinel_simple(
+		kWPANTUNDProperty_NCPCoexEnable,
+		SPINEL_PROP_RADIO_COEX_ENABLE, SPINEL_DATATYPE_BOOL_S);
```

Comment:
Same suggestion here to register with spinel capability check for coex (using `register_get_handler_capability_spinel_simple()`)

We can also add same capability check to metrics property

Reply:
Have updated.